src/Convex.jl

```diff
@@ -83,12 +86,9 @@ include("utilities/show.jl")
 include("utilities/iteration.jl")
 include("utilities/broadcast.jl")
 
-#Temporary workaround for memory leak (https://github.com/JuliaOpt/Convex.jl/issues/83)
+# Deprecated workaround for memory leak (https://github.com/JuliaOpt/Convex.jl/issues/83)
 function clearmemory()
```

Comment:
Given that `GC.gc()` should never be mandatory in user code as Convex, the function could be removed altogether

Reply:
Yeah, that's true. I thought it should at least be deprecated though, to give users time to remove it so they aren't hit with errors upon upgrading Convex if they have `Convex.clearmemory()` in their scripts. I wasn't sure about leaving in the `GC.gc()` or not; I agree they shouldn't need to be using it, but it was something that `Convex.clearmemory()` was doing, so I thought maybe I should point to it as a replacement in case somehow that's a key part of whatever they are doing.

Reply:
I changed the wording and removed the `GC.gc()` bit. I don't see any harm in leaving a deprecation message in for a month or two though, to let users comfortably remove the function without getting errors.